kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableWritesSuite.scala

```diff
@@ -92,21 +92,6 @@ abstract class AbstractDeltaTableWritesSuite extends AnyFunSuite with AbstractWr
     }
   }
 
-  test("create table - provide unsupported column types - expect failure") {
```

Comment:
Minor: Could we maybe create a custom data type `FooType extends DataType` and keep this test?

Reply:
Sure that's reasonable just to make sure we have the right error message

Reply:
yeah, also with this change, I think we never hit that error message at all since all types are now supported!